packages/fcl/src/current-user/index.js

```diff
@@ -102,8 +102,16 @@ async function authenticate(opts = {redir: false}) {
     const {discoveryWallet, discoveryWalletMethod, appDomainTag} =
       await buildAuthnConfig()
 
+    invariant(
+      service || discoveryWallet,
+      `
+        If no service passed to "authenticate," then "discovery.wallet" must be defined in config.
+        See: "https://github.com/onflow/flow-js-sdk/blob/master/packages/fcl/src/exec/query.md#configuration"
```

Comment:
May want to update this link to https://github.com/onflow/fcl-js/blob/267e0c4f160073a70ca8bb89008536fee7cf6b39/docs/reference/configure-fcl.mdx or add detail about discovery.wallet

Reply:
or your QS https://github.com/onflow/fcl-js/blob/b9d3dff55be5acdd739f29647f5059d0f8c69b84/docs/tutorials/flow-app-quickstart.mdx#configuration

Reply:
Good catch. Going to change it to this: https://docs.onflow.org/fcl/reference/api/#setting-configuration-values